Mathlib/Topology/ProperMap.lean

```diff
@@ -3,11 +3,11 @@ Copyright (c) 2023 Anatole Dedecker. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Anatole Dedecker, Etienne Marion
 -/
-
 import Mathlib.Topology.Homeomorph
 import Mathlib.Topology.StoneCech
 import Mathlib.Topology.Filter
 import Mathlib.Order.Filter.Cofinite
+import Mathlib.Topology.Defs.Sequences
```

Comment:
I don't think the imports need changing at all?

Reply:
This import is needed for the results mentioning `SequentialSpace`. This won't be necessary when `compactlyGeneratedSpace` typeclass is implemented.